tensorboard/plugins/graph/tf_graph_dashboard/tf-graph-dashboard.html

```diff
@@ -193,6 +200,17 @@
       value: true,  // TODO(@chihuahua): Reconcile this setting with Health Pills.
       readOnly: true,
     },
+    run: {
```

Comment:
Can you add a comment here noting the name of this property must coincide with the value of `storage.RUN`?

Reply:
Can rebase onto #509 to avoid this complexity.

Reply:
Done. Also checked that correct behavior was preserved (The URL changes when the run changes. The correct run is loaded when `run=` is set. A non-existent run triggers the dialog.).